src/main/resources/default/taglib/k/previewImage.html.pasta

```diff
@@ -8,29 +8,60 @@
 </i:pragma>
 
 <div class="col mb-4">
-    <div class="card card-border card-hover-shadow cursor-pointer w-100 @class">
-        <a onclick="openImageOverlay('@url')">
-            <div class="d-flex flex-grow-1 flex-column">
-                <div class="d-flex flex-grow-1 flex-row">
+    <div class="card w-100 @class">
+        <div class="d-flex flex-grow-1 flex-column">
+            <div class="d-flex flex-grow-1 flex-row">
+                <a onclick="openImageOverlay('@url')">
                     <img src="@url"
-                         class="img-fluid"
-                         style="height: 150px;width: 240px; object-fit: cover;"
+                         class="img-fluid cursor-pointer sci-preview-image sci-border-solid-1 sci-border-gray"
                          alt="Preview"/>
-                    <div class="w-75 m-2 flex-column">
-                        <p class="card-text sci-text-align-left">@title</p>
-                        <p class="card-text sci-text-align-left text-muted">@description</p>
-                    </div>
+                </a>
+                <div class="w-75 m-4 flex-column align-content-center">
+                    <p class="card-text sci-text-align-left sci-preview-image-title">@title</p>
+                    <p class="card-text sci-text-align-left text-muted">@description</p>
                 </div>
             </div>
-        </a>
+        </div>
     </div>
 </div>
 
 <script type="text/javascript">
     function openImageOverlay(imageUrl) {
-        const content = '<img src="' + imageUrl + '" class="img-fluid w-100" alt=""/>';
+        const content = '<div class="sci-overlay-content"><img src="' + imageUrl + '" class="img-fluid" alt=""/></div>';
 
         sirius.overlay.createOverlay({closable: true, yieldable: true, useFullScreen: true, content: content});
         sirius.overlay.showOverlay();
     }
 </script>
+
+<style>
+    .sci-preview-image-title {
+        font-size: 1.20rem;
+        font-weight: 500;
+    }
+
+    .sci-preview-image {
+        height: 110px;
+        width: 200px;
+        object-fit: contain;
+    }
+
+    .sci-overlay-content {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+        width: 100vw;
+
+    }
```

Comment:
Most of this should also be possible via Bootstrap CSS Classes like `d-flex`, ...
We should try to reduce the custom CSS as much as possible